pkgs/misc/preload/default.nix

```
@@ -0,0 +1,34 @@
+{ lib, stdenv, fetchurl, autoconf, automake, pkg-config, bash, glib }:
+
+stdenv.mkDerivation rec {
+  pname = "preload";
+  version = "0.6.4";
+
+  src = fetchurl {
+    url =
+      "http://downloads.sourceforge.net/sourceforge/${pname}/${pname}-${version}.tar.gz";
+    hash = "sha256-0KVY6DyymlHZ2Wc27zn0tOVeQ6WJrRrsWUoEjKIvgWs=";
+  };
+
+  patches = [ ./Makefile.patch ];
+
+  nativeBuildInputs = [ autoconf automake pkg-config ];
+  buildInputs = [ bash glib ];
```

`stdenv` [provides][1] `bash` 

```suggestion
  buildInputs = [ glib ];
```

[1]: https://nixos.org/manual/nixpkgs/unstable/#sec-tools-of-stdenv